Automattic/delphin
# Add login cookie

This pull request persists the login status of users so they are not forced to reconnect whenever they close their browser or open a new tab. More specifically, it:
- Saves the bearer token retrieved after verification in a cookie
- Retrieves the bearer token from the cookie on startup
- Fetches the user's profile on startup and hydrates the state tree with it
- Deletes the cookie upon logging out
#### Testing instructions
1. Run `git checkout add/login-cookie` and start your server
2. Open the [`Log In` page](http://delphin.localhost:1337/login)
3. Login and check that the menu now offers a `Log Out` option
4. Open a new tab, reload the page, or reopen your browser
5. Check that you still see the option in the menu
6. Logout and check that the option is gone

You can also check if the cookie is correctly removed when logging out.
#### Reviews
